dashboard/test/ui/support/hooks.rb

```diff
@@ -11,3 +11,11 @@
     saw_failure = true
   end
 end
+
+Before('@as_student') do
+  steps 'Given I manually sign in as "TestStudent"'
```

Comment:
Mehal, I think that sleep is already included in the [`I manually sign in as` step definition](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/step_definitions/steps.rb#L491).  Can you confirm that the 2-second wait there is what you're looking for?